### ISSUE TITLE
Glyph Window right click menu with Spiro

### ISSUE THREAD

**Issue by pathumego** [fontforge/fontforge#2261]
I expect to see 

```
G4 Curve
G2 Curve
Corner
Left Constraints
Right Constraints
```

on Right Click menu when Spiro is turned on. 
Instead it is showing the same `Curve`, `HV Curve`, `Corner`, `Tangent` that applies for beziers.

![screen shot 2015-05-06 at 10 47 59 pm](https://cloud.githubusercontent.com/assets/6846609/7499041/c475baca-f442-11e4-8056-dda48aaf5e97.png)

But as you can see the Point menu is changing accordingly.

![screen shot 2015-05-06 at 10 48 09 pm](https://cloud.githubusercontent.com/assets/6846609/7499055/e011d28c-f442-11e4-95cc-1f0258595a5e.png)

**Comment by pathumego** [comment]
@lpsandaruwan Please look in to this as well

**Comment by ctrlcctrlv** [comment]
This is indeed annoying, I've been trying to fix it recently, let's see if I can :-)

**Comment by ctrlcctrlv** [comment]
Wow - that was much easier to fix than I anticipated. To think I'd been using the "Get Info..." dialog all this time instead because I feared touching the code for the menu!! I've opened a PR with my fix,  #3581.

![g4etc](https://user-images.githubusercontent.com/838783/54876900-a429d300-4e52-11e9-9b5c-67bb403b1d26.png)
